src/test/mir-opt/inline/caller-with-trivial-bound.rs

```diff
@@ -1,5 +1,5 @@
+// ignore-wasm32 compiled with panic=abort by default
```

Comment:
Please also add `// needs-unwind` if `-Cpanic=unwind` is necessary. Otherwise this test will fail for cg_clif, which forces `-Cpanic=abort` right now. (I should probably make compiletest check for `panic="abort"` in the `rustc --print cfg` output to set the `panic_abort` flag that causes `// needs-unwind` tests to be skipped. That would remove the need for `// ignore-wasm32` when `// needs-unwind` is already used.